deploy/.env.ingest.production.local

```diff
@@ -0,0 +1,3 @@
S29vV0VXVndIR24yeVIzNmdLTG96bWI0WWpESkdlcm90QVBHeG1kV1p4Mm54TUM0IiwKICAgICAgIi9kbnM0L2Jvb3RzdHJhcC0xLnN0YXJwb29sLmluL3RjcC8xMjc1Ny9wMnAvMTJEM0tvb1dRWnJHSDFQeFNOWlB1bTk5TTF6TnZqTkZNMzNkMUFBdTVEY3ZkSHB0dVU3dSIsCiAgICAgICIvZG5zNC9ib290c3RyYXAtMC5tYWlubmV0LmZpbG9wcy5uZXQvdGNwLzEzNDcvcDJwLzEyRDNLb29XQ1ZlOE1tc0VNZXMyRnpnVHB0OWZYdG1DWTd3cnE5MUdSaWFDOFBIU0NDQmoiLAogICAgICAiL2RuczQvYm9vdHN0cmFwLTEubWFpbm5ldC5maWxvcHMubmV0L3RjcC8xMzQ3L3AycC8xMkQzS29vV0N3ZXZIZzF5TEN2a3RmMm52THU3TDk4OTRtY3JKUjRNc0JDY200c3lTaFZjIiwKICAgICAgIi9kbnM0L2Jvb3RzdHJhcC0wLnN0YXJwb29sLmluL3RjcC8xMjc1Ny9wMnAvMTJEM0tvb1dHSHBCTWVaYmVzdFZFV2tmZG5DOXU3cDZ1RkhYTDFuN20xWkJxc0VtaVV6eiIsCiAgICAgICIvZG5zNC9sb3R1cy1ib290c3RyYXAuaXBmc2ZvcmNlLmNvbS90Y3AvNDE3NzgvcDJwLzEyRDNLb29XR2h1Zk5tWkhGM3N2NDhhUWVTMTNuZzVYVkpaOUU2cXkyTXM0VnpxZVVzSGsiLAogICAgICAiL2RuczQvYm9vdHN0cmFwLTEuaXBmc21haW4uY24vdGNwLzM0NzIzL3AycC8xMkQzS29vV01LeE1rRDVETXBTV3NXN2RCZGRLeEtUN0wyR2diTnVja3o5b3R4dmt2QnlQIgogICAgXSwKICAgICJNaW5pbXVtUGVlcnMiOiA0CiAgfSwKICAiRGF0YXN0b3JlIjogewogICAgIlR5cGUiOiAiZHluYW1vZGIiLAogICAgIkRpciI6ICJ2aWMtaXBuaS1kYXRhc3RvcmUiLAogICAgIlJlZ2lvbiI6ICJ1cy13ZXN0LTIiLAogICAgIlRtcFR5cGUiOiAiZHluYW1vZGIiLAogICAgIlRtcERpciI6ICJ2aWMtaXBuaS10bXAtZGF0YXN0b3JlIiwKICAgICJUbXBSZWdpb24iOiAidXMtd2VzdC0yIgogIH0sCiAgIkRpc2NvdmVyeSI6IHsKICAgICJGaWx0ZXJJUHMiOiB0cnVlLAogICAgIklnbm9yZUJhZEFkc1RpbWUiOiAiMmgwbTBzIiwKICAgICJQb2xpY3kiOiB7CiAgICAgICJBbGxvdyI6IHRydWUsCiAgICAgICJFeGNlcHQiOiBudWxsLAogICAgICAiUHVibGlzaCI6IHRydWUsCiAgICAgICJQdWJsaXNoRXhjZXB0IjogbnVsbAogICAgfSwKICAgICJQb2xsSW50ZXJ2YWwiOiAiMjRoMG0wcyIsCiAgICAiUG9sbFJldHJ5QWZ0ZXIiOiAiNWgwbTBzIiwKICAgICJQb2xsU3RvcEFmdGVyIjogIjE2OGgwbTBzIiwKICAgICJQb2xsT3ZlcnJpZGVzIjogbnVsbCwKICAgICJVc2VBc3NpZ25lciI6IGZhbHNlCiAgfSwKICAiSW5kZXhlciI6IHsKICAgICJDYWNoZVNpemUiOiAtMSwKICAgICJDb25maWdDaGVja0ludGVydmFsIjogIjMwcyIsCiAgICAiU2h1dGRvd25UaW1lb3V0IjogIjE1bSIsCiAgICAiVmFsdWVTdG9yZVR5cGUiOiAiZHluYW1vZGIiLAogICAgIkR5bmFtb0RCUmVnaW9uIjogInVzLXdlc3QtMiIsCiAgICAiRHluYW1vREJQcm92aWRlcnNUYWJsZSI6ICJ2aWMtaXBuaS12YWx1ZXN0b3JlLXByb3ZpZGVycyIsCiAgICAiRHluYW1vREJNdWx0aWhhc2hNYXBUYWJsZSI6ICJ2aWMtaXBuaS12YWx1ZXN0b3JlLW11bHRpaGFzaC1tYXAiLAogICAgIkZyZWV6ZUF0UGVyY2VudCI6IC0xCiAgfSwKICAiSW5nZXN0IjogewogICAgIkFkdmVydGlzZW1lbnREZXB0aExpbWl0IjogMzM1NTQ0MzIsCiAgICAiRW50cmllc0RlcHRoTGltaXQiOiA2NTUzNiwKICAgICJIdHRwU3luY1JldHJ5TWF4IjogMCwKICAgICJIdHRwU3luY1JldHJ5V2FpdE1heCI6ICIzMHMiLAogICAgIkh0dHBTeW5jUmV0cnlXYWl0TWluIjogIjFzIiwKICAgICJIdHRwU3luY1RpbWVvdXQiOiAiMTBzIiwKICAgICJJbmdlc3RXb3JrZXJDb3VudCI6IDEwLAogICAgIlB1YlN1YlRvcGljIjogIi9pbmRleGVyL2luZ2VzdC9tYWlubmV0IiwKICAgICJSZXNlbmREaXJlY3RBbm5vdW5jZSI6IGZhbHNlLAogICAgIlN5bmNTZWdtZW50RGVwdGhMaW1pdCI6IDIwMDAsCiAgICAiU3luY1RpbWVvdXQiOiAiMmgwbTBzIgogIH0sCiAgIkxvZ2dpbmciOiB7CiAgICAiTGV2ZWwiOiAiaW5mbyIsCiAgICAgICJMb2dnZXJzIjogewogICAgICAiYmFzaWNob3N0IjogIndhcm4iLAogICAgICAiYm9vdHN0cmFwIjogIndhcm4iLAogICAgICAiZHQtaW1wbCI6ICJ3YXJuIiwKICAgICAgImR0X2dyYXBoc3luYyI6ICJ3YXJuIiwKICAgICAgImdyYXBoc3luYyI6ICJ3YXJuIgogICAgfQogIH0sCiAgIlBlZXJpbmciOiB7CiAgICAiUGVlcnMiOiBudWxsCiAgfQp9Cgo=
```

Comment:
Did we mean to push this? Looks like it was generated from deploy/.env.ingest.production.local.tpl?

Reply:
good catch, thanks. I removed the file and added the corresponding ignores.